tests/lib.rs

```diff
@@ -110,6 +110,7 @@ mod timestamps;
 mod undefined_variables;
 mod unexport;
 mod unstable;
+mod which_exec;
```

Comment:
```suggestion
mod which;
```

Reply:
Unfortunately this doesn't work because of the name conflict with the `which` crate, which is imported and used in this test suite:

https://github.com/casey/just/blob/4f31853835afcd8d511fccf7b2b6680f6a628446/tests/lib.rs#L28

Reply:
Ahh, gotcha

Reply:
I'm happy to rename it from `which_exec` to something, e.g., `which_fn`, if you prefer. I chose `which_exec` to evoke something like "`which_exec`utable is `cmd`"